src/langchain_google_alloydb_pg/async_vectorstore.py

```diff
@@ -541,6 +541,19 @@ async def __query_collection(
         operator = self.distance_strategy.operator
         search_function = self.distance_strategy.search_function
 
+        columns = self.metadata_columns + [
+            self.id_column,
+            self.content_column,
+            self.embedding_column,
+        ]
+        if self.metadata_json_column:
+            columns.append(self.metadata_json_column)
+
+        if columns:
+            column_names = " ,".join(columns)
```

Comment:
Does it make sense to do like this instead? This could help if user provides something like "column-name."

```suggestion
            column_names = ", ".join(f'"{col}"' for col in columns)
```

Reply:
I like the extra type safety.